docs/RUNNING.md

```diff
@@ -65,7 +65,7 @@ First, let's try to fetch some repository data and see if it works.
 No database is needed at this point.
 
 ```
-./repology-update --fetch --parse
```

Comment:
This change is not correct. The idea is that ```family``` condition correspond to ```family``` field from repository definitions. schemacheck is a derivative from that.